androidApp/src/main/java/com/mbta/tid/mbta_app/android/stopDetails/AlertCard.kt

```diff
@@ -76,23 +76,13 @@ fun AlertCard(
             AlertIcon(
                 alertState = alert.alertState,
                 color = color,
-                modifier =
-                    Modifier.size(iconSize)
-                        .align(
-                            if (spec == AlertCardSpec.Elevator) Alignment.Top
-                            else Alignment.CenterVertically
-                        )
+                modifier = Modifier.size(iconSize).align(Alignment.CenterVertically)
```

Comment:
One other minor thing - unfortunately the NotAccessibleCard is separate from this and will need to be updated as well for vertical icon centering